.travis.yml

```diff
@@ -13,11 +13,6 @@ cache:
 env:
   # we recommend new addons test the current and previous LTS
   # as well as latest stable release (bonus points to beta/canary)
-  - EMBER_TRY_SCENARIO=ember-lts-2.8
```

Comment:
We still need to revert these changes, and we really should add `ember-1.11` too but we can get this merged for now